Documentation/Development.md

```diff
@@ -218,19 +290,15 @@ cp swift-apis/Sources/x10/swift_bindings/xla_tensor_wrapper.h ${DESTDIR}/usr/inc
 ```
 </details>
 
-### (TensorFlow) Swift APIs
+### Building Swift APIs
 
-#### SwiftPM
+> **Note:** This option requires pre-built X10 binaries. Building with SwiftPM
+> does not include changes to X10.
 
-*Note: Building with SwiftPM does not include changes to X10 modules.*
+SwiftPM requires two items:
 
-Because X10 is a required component for building the TensorFlow Swift APIs,
-and the project does not build with Swift Package Manager, we must pass along
-the necessary information to the build. We need to pass two items to the
-build:
-
-1. the location of the X10 & TensorFlow headers
-2. the location of the X10 & TensorFlow libraries
+1. The location of the X10 & TensorFlow headers.
+2. The location of the X10 & TensorFlow libraries.
```

Comment:
One thing that I feel is nice about the CMake section is that we explicitly indicate the command which is for testing, and which is for building.  Do you think it is valuable to replicate that in the SPM section below as well?